site/reference/cli/fn/eval/README.md

```diff
@@ -130,6 +130,10 @@ fn-args:
   it doesn't exist. Structured results emitted by the functions are aggregated and saved
   to `results.yaml` file in the specified directory.
   If not specified, no result files are written to the local filesystem.
+
+--safe:
+  Safe the function image and fn-config to Kptfile. Require `--image`.
```

Comment:
Few comments:

- display the short flag as well. I am guessing `-s` is the short flag
- Instead of saving `Require --image`, may be, we should specify the behavior that declarative form is saved only if the function is successfully evaluated. That should cover all the validity requirements for the command.
- I think we can improve the text. I think on a high level, this flag turns the imperative invocation of the function into declarative form and adds to `pipeline` of the package.

Reply:
done.